packages/example-forms/lib/modules/customers/schema.js

```diff
@@ -0,0 +1,111 @@
+/*
+
+A SimpleSchema-compatible JSON schema for prospects
+
+*/
+import SimpleSchema from 'simpl-schema'
+import { Components } from 'meteor/vulcan:core'
+import { addressFormSchema } from '../address'
+import ArrayForm from '../../components/ArrayForm'
+
+// Defining the allowed value for the state field
+// Using separate object is a good practice, since you might need those  info
+// elsewhere in your app (e.g when displaying the data in a component)
+export const stateAllowedValues = ['prospect', 'client', 'scheduled']
+export const stateLabels = {
+    prospect: "Prospect",
+    client: "Client",
+    scheduled: "Meeting scheduled",
+}
+
+const schema = {
+    _id: {
+        type: String,
+        optional: true,
+        viewableBy: ['guests'],
+    },
+    // document creation date is set automatically (optional + hidden + onInsert)
+    createdAt: {
+        type: Date,
+        optional: true,
+        hidden: true,
+        onInsert: (document, currentUser) => {
+            return new Date();
+        }
+    },
+    // Who created the doc. This field is necessary for Vulcan to handle ownership
+    // correctly when setting the permissions
+    userId: {
+        type: String,
+        hidden: true,
+        optional: true,
+        onInsert(document, currentUser){
+            document.userId = currentUser._id
+        },
+    },
+    state: {
+        type: String,
+        allowedValues: stateAllowedValues,
+        default: 'prospect',
+        optional: false,
+        searchable: true,
+        form: {
```

Comment:
Using `form` is more or less deprecated, I think it's better to put `options` directly on the root of the field. For passing props to the field specifically there is `fieldProperties`.

Reply:
@sacha is that a SmartForm thing or SimpleSchema thing?

Reply:
A SmartForm thing, SimpleSchema doesn't know/care about forms.